services/api-server/src/simcore_service_api_server/services/webserver.py

```diff
@@ -244,11 +244,13 @@ async def update_me(self, profile_update: ProfileUpdate) -> Profile:
     # PROJECTS -------------------------------------------------
 
     @_exception_mapper({})
-    async def create_project(self, project: ProjectCreateNew) -> ProjectGet:
+    async def create_project(
+        self, project: ProjectCreateNew, hidden: bool
```

Comment:
MINOR: rename `is_hidden` and enforce key argument
```python
async def foo(self, project: ProjectCreateNew, *, is_hidden: bool)
    ...
```

Reply:
yep, good point. Done